rules/dones-do-your-dones-include-a-url/rule.md

```diff
@@ -40,43 +40,53 @@ Done - northwind&#46;com/about-us
 
 :::
 ::: good
-Figure: Good example - Easy to check what was done 
+Figure: Good example - Easy to check what was done
 :::
 
 ::: greybox
-Done - northwind&#46;com/about-us/ as requested on ssw2&#46;visualstudio&#46;com/Northwind/\_workitems/edit/00001 
+Done - northwind&#46;com/about-us/ as requested on ssw2&#46;visualstudio&#46;com/Northwind/\_workitems/edit/00001
```

Comment:
```suggestion
Done - northwind.com/about-us/ as requested on ssw2.visualstudio.com/Northwind/\_workitems/edit/00001
```

Please update the rest of these unicode strings to `.` periods

Reply:
Hi @JackDevAU 

We don't want these to be clickable and give 404s, that's why we used unicode